usb.cpp

```diff
@@ -124,7 +124,9 @@ namespace bsp
     void usbUnlockMTP()
     {
         log_debug("mtpUnlock");
+#if defined(USB_DEVICE_CONFIG_MTP) && (USB_DEVICE_CONFIG_MTP > 0U)
```

Comment:
Maybe the log should also be moved inside the `if`? Seeing the log with MTP disabled might be a little bit confusing IMO.

Reply:
You have totally right! I'll change it. Thanks